utilities/_layout.scss

```diff
@@ -0,0 +1,36 @@
+//Full responsive layout based on design system
+//Todo: amend spacing calc for between 414px and 599 as some misalignment
+
+
```

Comment:
I think this part needs to live in the component library as the spacing etc is specific to the redesign

Reply:
Have amended and moved to component library